MTM-60356 - changelog for a bugfix regarding capability to create measurement series without value (#2459)
SoftwareAG/c8y-docs: content/change-logs/platform-services/cumulocity-10-20-566-0-fixed-measurement-series-without-value.md

---

> date: 2024-09-23

Reviewer: ```suggestion
date: 
```

> title: Measurement series without value can no longer be created

Reviewer: ```suggestion
title: Measurement series can no longer be created without value
```

> title: Fixed the capability to create measurement series without value

Reviewer 2: ```suggestion
title: Measurement series can now be created without value
```

Author: I changed wording a little bit. Now such measurements can no longer be created.

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-OG_650_b2
    label: Core platform
build_artifact:
  - value: tc-QHwMfWtBk7
    label: cumulocity
ticket: MTM-60356
version: 10.20.566.0
---

> Fixed a regression in time series measurements that allowed to create a measurement series without value.

Reviewer: ```suggestion
Previously, it was possible to create a measurement series without value. This issue has been fixed as the value is mandatory. Now a value must be provided on measurement creation. 
```

> Fixed a regression in time series measurements that allowed to create a measurement series without value.

Reviewer 2: Please add previous status.

Author: What do you exactly mean by 'previous status'? How it used to work before the change?

Previously, it was possible to create measurement without value which was discrepant with the [API documentation](https://www.cumulocity.com/api/core/#operation/postMeasurementCollectionResource) (`The value is mandatory for a fragment.`)
